src/app/effects/book.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { EffectsTestingModule, EffectsRunner } from '@ngrx/effects/testing';
+import {TestBed, fakeAsync, tick} from '@angular/core/testing';
+import {BookEffects} from './book';
+import {GoogleBooksService} from '../services/google-books';
+import {Observable} from 'rxjs';
+import {SearchAction, SearchCompleteAction} from '../actions/book';
+import {Book} from '../models/book';
+
```

Comment:
@krjordan

So I downloaded Atom to make sure the typing issue was fixed. Seems to be just an Atom issue as VSCode and Webstorm have no issue. Your fix does work, I suggest we put it in /src/test.ts instead of in each file as the functions should be global for all tests. 

**/src/test.ts**
```
import {} from 'jasmine';
```

This worked for me. What do you think?